.vscode/launch.json

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
why tho, is this so u can run the app through the GUI? or for debugging? cause for debugging you can just run any node process in VSCode's debug terminal

also - make sure u open the workspace through the workspace.code-workspace file for all of the configs to be properly applied, instead of just opening the folder

Reply:
Mainly because I like pressing a GUI button.   I can walk you through what I'm doing